contracts/PNSResolver.sol

```diff
@@ -65,4 +64,25 @@ contract PNSResolver is Initializable, AddrResolver, OwnableUpgradeable {
 	function isAuthorised(bytes32 phoneHash) internal view override returns (bool) {
 		return msg.sender == getOwner(phoneHash);
 	}
+
+	function seedResolver(bytes32 phoneHash, address a) external registryAuthorised(phoneHash) {
+		seedAddr(phoneHash, COIN_TYPE_ETH, addressToBytes(a));
+	}
+
+	function seedAddr(
+		bytes32 phoneHash,
+		uint256 coinType,
+		bytes memory a
+	) internal registryAuthorised(phoneHash) {
+		emit AddressChanged(phoneHash, coinType, a);
```

Comment:
Is there a particular reason we're using an arbitrary name like `a`? I feel like it's not descriptive enough

Reply:
can be change currently just a direct copy of ENS